src/app/me/page.tsx

```diff
@@ -155,101 +158,140 @@ const Content = () => {
               value={currentUser?.email ?? ""}
               onChange={() => {}}
               border="1px solid #888"
-              minWidth="320px"
+              width="320px"
               isDisabled={true}
             />
           </FormControl>
         </Box>
       </Flex>
-      <Flex gap="1rem">
+      <Flex direction="column" gap="1rem" mt={3}>
+        <Heading size="md">Vectara Credentials</Heading>
         <Box>
-          <FormControl style={formControlStyles}>
-            <FormLabel style={formLabelStyles}>Vectara Customer ID</FormLabel>
-            <Input
-              type="text"
-              value={formState.vectaraCustomerId ?? ""}
-              onChange={(e: ChangeEvent<HTMLInputElement>) => {
-                setFormState({
-                  ...formState,
-                  vectaraCustomerId: e.target.value ?? undefined,
-                });
-
-                setIsSubmitedDisabled(false);
-              }}
-              border="1px solid #888"
-              minWidth="320px"
-            />
-          </FormControl>
+          <Text fontSize="xs" color="#FEFCBF" fontWeight={600}>
+            Don't have a Vectara account?{" "}
+            <Link color="blue.500" href="https://console.vectara.com/signup">
```

Comment:
Please use: https://console.vectara.com/signup/?utm_source=App&utm_medium=Portal&utm_term=sign-up&utm_content=sign-up-in-portal&utm_campaign=App-Portal-sign-up-sign-up-in-portal